request/en.txt

```diff
@@ -0,0 +1,65 @@
+Dear ○○,
+
+This is a staff of a Ruby 25th anniversary event.
+Ruby was born in 24th Feb, 1993, so will be 25th birthday.
+We held a event that celebrating and talking about Ruby's past, now and future, on 24th Feb. 2018, in Tokyo.
+
+We're also make a event on web collecting messages for Ruby from over the world Rubyist.We'd like to ask you to write message for Ruby and Rubyist, though I know how busy you are. We are planning to open the messages web pages on 1st Feb.
+
+## About the message
+
+About Ruby
+We don't make constraint for detail
+It's good to write past, now and future.
+License: CC BY-SA
+https://creativecommons.org/licenses/by-sa/2.0/
+
+And we make code of conduct in the event, so the message is also under the that.
+*** TODO: CoC url
+
+## Format
+
+There're 2 ways.
+
+### 1. GitHub Flavored Markdown
+
+GitHub Flavored Markdown: https://guides.github.com/features/mastering-markdown/
+Please place big files like an image or movie outside of repository, to prevent repository size will be huge.
```

Comment:
Please place big files like an image or movie outside of the repository, to prevent repository size from getting huge.